devtools/conda-envs/test-env.yaml

```diff
@@ -41,6 +41,7 @@ dependencies:
   - chemper
   - geometric
   - torsiondrive
+  - pymbar <4
```

Comment:
Great catch. Hopefully this should be handled by the pins in the new ForceBalance build, but this is a good way to be sure.